.github/workflows/ci.yml

```diff
@@ -14,6 +14,7 @@ jobs:
         machine: [ windows-2022, ubuntu-20.04, macos-10.15 ]
     runs-on: ${{ matrix.machine }}
     steps:
+      - uses: microsoft/setup-msbuild@v1.1
```

Comment:
```suggestion
      - uses: microsoft/setup-msbuild@v1.1.0
```

Reply:
Addressed https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/401/commits/86cd11714e739c284d26809bff56f1cbff51a7af

Reply:
It occurred to be good... https://github.com/microsoft/setup-msbuild/releases/tag/v1.1